docs/notebooks/qubit_mapping_guide.ipynb

```diff
@@ -0,0 +1,796 @@
+{
```

Comment:
Mapomatic is included in Qiskit transpiler's present pass managers (as [VF2Layout](https://docs.quantum.ibm.com/api/qiskit/qiskit.transpiler.passes.VF2Layout) or [VF2PostLayout](https://docs.quantum.ibm.com/api/qiskit/qiskit.transpiler.passes.VF2PostLayout)) for all `optimization_level`s except `0`. 

I think we should mention it somewhere in the notebook. Maybe, we can borrow the rationale befind using standaalone `mapomatic` from [mapomatic's GitHub](https://github.com/qiskit-community/mapomatic?tab=readme-ov-file#qiskit-transpiler).